week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,62 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+let possibleAnswers = ["It is certain",
+  "It is decidedly so",
+  "Without a doubt",
+  "Yes - definitely",
+  "You may rely on it",
+  
+  "As I see it, yes",
+  "Most likely",
+  "Outlook good",
+  "Yes",
+  "Signs point to yes",
+  
+  "Reply hazy, try again",
+  "Ask again later",
+  "Better not tell you now",
+  "Cannot predict now",
+  "Concentrate and ask again",
+  
+  "Don't count on it",
+  "My reply is no",
+  "My sources say no",
+  "Outlook not so good",
+  "Very doubtful"];
+
+
+function shakeBall() {
+  let answer = possibleAnswers[Math.floor(Math.random()* possibleAnswers.length)] ;
+  console.log(`The ball has shaken!`);
+  return answer;
+}
```

Comment:
Clean and concise 👌

Good job too at using `possibleAnswers.length` instead of just using `20`.